driver/csiplugin/utils/http_utils.go

```diff
@@ -78,7 +84,13 @@ func HttpExecuteUserAuth(ctx context.Context, httpClient *http.Client, requestTy
 	request.Header.Add("Accept", "application/json")
 
 	request.SetBasicAuth(user, password)
-	klog.V(6).Infof("[%s] http_utils HttpExecuteUserAuth request: %+v", GetLoggerId(ctx), request)
+
+	logLevel := strings.ToUpper(os.Getenv(LogLevel))
+	if strings.Contains(requestURL, BucketKeysURL) && logLevel == TRACE.String() && request != nil {
```

Comment:
if we restrict the condition to AFM cache feature then for normal cases we might not see this log.

Reply:
Right, fixed it

Reply:
@saurabhwani5, the driver image with this log change is updated [here](https://github.com/IBM/ibm-spectrum-scale-csi/pull/1181#issuecomment-2217235612)